packages/cashscript/test/Contract.test.ts

```diff
@@ -1,6 +1,6 @@
 import { hexToBin } from '@bitauth/libauth';
 import { placeholder } from '@cashscript/utils';
-import { Contract, ElectrumNetworkProvider, SignatureTemplate } from '../src';
```

Comment:
Why does this need to change from `src/` to `dist/`? One nice thing was being able to run `jest` without needing to recompile first, is that still the case, or does this require me to compile before testing?

Reply:
Only importing from the module entrypoint doesn't help in this case right? Since there's still the difference between `../src/index.js` and `../dist/main/index.js`.

Tests weren't running correctly with the `../dist/main/...` updates, so I changed it back to `../src/...` as well as adding some additional config that may help fix the issues you were seeing before. 

I also added file extensions to some other places where you hadn't added it yet.

Can you try again and see if it's working for you?

Re incremental compilation, definitely something that sounds useful and I'd like to add. But I'd prefer to add that separately after getting this stuff to work and after getting the `next` code merged into main.

Reply:
Right, two different issues – seems like that config change works! And I just confirmed that the package works when linked and used in the patch I'm working on here 👍: https://gitlab.com/GeneralProtocols/anyhedge/library/-/merge_requests/193